osidb/tests/test_flaw.py

```diff
@@ -795,6 +797,138 @@ def test_cvss_rh_nvd_severity_diff_valid(self, vector_pair):
+                "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",  # score 8.1
+                "rhel-7",
+                "",
+                Flaw.FlawNistCvssValidation.NOVALUE,
+                False,
+            ),
+            # NIST CVSS and RH CVSS are not significantly different
+            (
+                "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:L/A:H",  # score 7.1
+                "CVSS:3.1/AV:L/AC:H/PR:N/UI:R/S:U/C:H/I:H/A:H",  # score 7.0
+                "rhel-7",
+                "",
+                Flaw.FlawNistCvssValidation.NOVALUE,
+                False,
+            ),
+            # no RH product
+            (
+                "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:L/A:H",  # score 7.1
+                "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",  # score 8.1
+                "rhel-br-9",
+                "",
+                Flaw.FlawNistCvssValidation.NOVALUE,
+                False,
+            ),
+            # solved via RH CVSS comment
+            (
+                "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:L/A:H",  # score 7.1
+                "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",  # score 8.1
+                "rhel-7",
+                "explanation comment",
+                Flaw.FlawNistCvssValidation.NOVALUE,
+                False,
+            ),
+            # solved via NIST rescore request
+            (
+                "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:L/A:H",  # score 7.1
+                "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",  # score 8.1
+                "rhel-7",
+                "",
+                Flaw.FlawNistCvssValidation.REQUESTED,
+                False,
+            ),
+        ],
+    )
+    def test_validate_nist_rh_cvss_feedback_loop(
+        self,
+        nist_cvss,
+        rh_cvss,
+        rh_cvss_comment,
+        ps_module,
+        rescore,
+        should_alert,
+    ):
+        """
+        Tests whether RH should send a request to NIST on flaw CVSS rescore.
+        """
+        flaw = FlawFactory(
+            nist_cvss_validation=rescore,
+            # fields below are set to avoid any alerts
+            embargoed=False,
+            major_incident_state=Flaw.FlawMajorIncident.NOVALUE,
+            summary="summary",
+            statement="statement",
+        )
+        FlawCVSSFactory(
+            flaw=flaw,
+            version=FlawCVSS.CVSSVersion.VERSION3,
+            issuer=FlawCVSS.CVSSIssuer.NIST,
+            vector=nist_cvss,
+            comment="",
+        )
+        FlawCVSSFactory(
+            flaw=flaw,
+            version=FlawCVSS.CVSSVersion.VERSION3,
+            issuer=FlawCVSS.CVSSIssuer.REDHAT,
+            vector=rh_cvss,
+            comment=rh_cvss_comment,
+        )
+        PsModuleFactory(
+            name=ps_module, ps_product=PsProductFactory(business_unit="Core RHEL")
+        )
+        AffectFactory(flaw=flaw, ps_module=ps_module)
+        flaw.save(raise_validation_error=False)
```

Comment:
I wonder why do we have to suppress the validation errors here. The tested validation does not raise so what is it we cannot just setup not by the factories not to raise any exceptions?

Reply:
`raise_validation_error=False` should not be here, I will remove it.